eurocodedesign/units/__init__.py

```diff
@@ -0,0 +1,255 @@
+from abc import ABC
+from enum import Enum, unique, auto
+from functools import partial
+from typing import Self, TypeAlias
+
+
+@unique
+class PhysicalType(Enum):
+    DIMENSIONLESS = auto()
```

Comment:
Can `DIMENSIONLESS` be removed?
`PhysicalType` is a property of the units and by definition if you have a unit the `PhysicalType` can't be dimensionless

Reply:
Okay, I will remove angle, too.